.env

```
@@ -25,7 +25,10 @@ ENABLE_LATENCY_CONTROL=0
 # GAME_TYPE=openarena
 
 # Dota 2
-GAME_TYPE=dota2
-DOTA2_RCON_HOST=10.10.10.34
-DOTA2_RCON_PORT=27015
-DOTA2_RCON_PASSWORD=XlB*IAF21!Kx
+GAME_TYPE=amp
+
+AMP_BASE_URL=https://amp.dotnetbistro.cloud
+AMP_USERNAME=minh
+AMP_PASSWORD=nbz!KPX-tvg-yhz9xvy
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AMP_PASSWORD key should go before the AMP_USERNAME key

(UnorderedKey)